sdk/core/azure-core-amqp/src/impl/rust_amqp/rust_amqp/azure_core_amqp/README.md

```diff
@@ -8,7 +8,7 @@ This crate is part of a collection of crates: for more information please refer
 
 ## Example
 
-```no_run,rust
+```rust,no_run
```

Comment:
I've never seen the comma documented. Maybe it works, but with commonmark and other renders the delimiter is typically just a space:
```suggestion
```rust no_run
```